packages/server/package.json

```diff
@@ -99,7 +99,7 @@
     "node-fetch": "2.6.7",
     "open": "8.4.0",
     "pg": "8.5.1",
-    "pino-pretty": "4.0.0",
+    "pino-pretty": "5.1.3",
```

Comment:
This was causing a strange build issue. The "breaking change" between 4 and 5 is that 5 adds TS types